src/local/object_id/raw.rs

```diff
@@ -250,6 +286,29 @@ mod test {
         }
     }
 
+    #[test]
```

Comment:
Following @Dessix suggestion, it might be worth testing on wasm.

Reply:
I believe CI _should_ be running all plain `#[test]` tests on WASM too as of https://github.com/rustyscreeps/screeps-game-api/commit/a1d21625896a8b567b1e05f623062d858b320788.

It doesn't seem to indicate what tests succeed, but a cursory test shows that locally, `cargo web test --verbose --nodejs` will fail if I insert a manual panic in here.